doc/man7/provider-cipher.pod

```diff
@@ -218,13 +218,24 @@ For example AES in CTR mode has a block size of 1 (because it operates like a
 stream cipher), even though AES has a block size of 16.
 The length of the "blocksize" parameter should not exceed that of a B<size_t>.
 
-=item "flags" (B<OSSL_CIPHER_PARAM_FLAGS>) <unsigned integer>
+=item "aead" (B<OSSL_CIPHER_PARAM_AEAD>) <integer>
+Gets 1 if this is an AEAD cipher algorithm, otherwise it returns 0.
```

Comment:
Does this need a blank line above?
Also below.

Reply:
ta